File: LangtonsAnt/Square.cpp
#include "Square.h"

int Square::x() {
	return _x;
}

int Square::y() {
	return _y;
}

char Square::color() {
	return _color;
}

Square::Square(int x, int y, char color) {
	_x = x;
	_y = y;
	_color = color;
}

File: LangtonsAntCSharp/Models/Square.cs
namespace Models
{
    public class Square
    {
        public int X { get; set; }
        public int Y { get; set; }
        public bool IsWhite { get; set; }
        Square() { }
        public Square(int x, int y)
        {
            this.X = x;
            this.Y = y;
            this.IsWhite = true;
        }
        public Square(int x, int y, bool isWhite = true)
        {
            this.X = x;
            this.Y = y;
            this.IsWhite = isWhite;
        }
    }
}
